### PR TITLE
Added official repositories to install guide.

### PR DESCRIPTION
**Pull request by petrkotas**
Added required repositories to install ovirt engine install guide. Without these repositories the instalation guide will not work.
-

-

-

I confirm that this pull request was submitted according to the [contribution guidelines](/CONTRIBUTING.md): (please @petrkotas yourself to sign)

---
This change is [Reviewable](https://reviewable.io/reviews/ovirt/ovirt-site/1112)

### CODE REVIEW

File: source/documentation/install-guide/chap-Installing_oVirt.html.md [hunk 1]
```diff
@@ -10,13 +10,21 @@ Before you can configure and use the oVirt Engine, you must install the <package
 
 **Installing the oVirt Engine Packages**
 
-1. To ensure all packages are up to date, run the following command on the machine where you are installing the oVirt Engine:
+1. Before you can start installing the oVirt, add the official repository. Choose one based on your version requirement:
+
+        
+        # http://resources.ovirt.org/pub/yum-repo/ovirt-release35.rpm
```
Comment (mykaul) [inline source/documentation/install-guide/chap-Installing_oVirt.html.md]: I'd order them differently. Current (4.1) first, then older releases, from 4.0 to 3.5.

File: source/documentation/install-guide/chap-Installing_oVirt.html.md [hunk 2]
```diff
@@ -10,13 +10,21 @@ Before you can configure and use the oVirt Engine, you must install the <package
 
 **Installing the oVirt Engine Packages**
 
-1. To ensure all packages are up to date, run the following command on the machine where you are installing the oVirt Engine:
+1. Before you can start installing the oVirt, add the official repository. Choose one based on your version requirement:
```
Comment (mykaul) [inline source/documentation/install-guide/chap-Installing_oVirt.html.md]: 'the oVirt' -> 'oVirt'

File: source/documentation/install-guide/chap-Installing_oVirt.html.md (hunk 1 above)
Comment (petrkotas) [inline source/documentation/install-guide/chap-Installing_oVirt.html.md]: Thanks, this makes perfect sense. If this is now OK, can we merge this pull request?
It has the potential to save people from struggling.

Comment (mykaul) [inline source/documentation/install-guide/chap-Installing_oVirt.html.md]: @petrkotas - can you push the changes that respond to my review comments?

Comment (petrkotas) [inline source/documentation/install-guide/chap-Installing_oVirt.html.md]: Yes my bad, I have read that as You did that.

**Comment by mykaul** [comment]
@JohnMarksRH - looks good to me. Please review and merge.

File: source/documentation/install-guide/chap-Installing_oVirt.html.md [hunk 3]
```diff
@@ -10,13 +10,21 @@ Before you can configure and use the oVirt Engine, you must install the <package
...
+
+        
+        # http://resources.ovirt.org/pub/yum-repo/ovirt-release41.rpm
+        # http://resources.ovirt.org/pub/yum-repo/ovirt-release40.rpm
+        # http://resources.ovirt.org/pub/yum-repo/ovirt-release36.rpm
+        # http://resources.ovirt.org/pub/yum-repo/ovirt-release35.rpm
+
+2. To ensure all packages are up to date, run the following command on the machine where you are installing oVirt Engine:
```
Comment (JohnMarksRH) [inline source/documentation/install-guide/chap-Installing_oVirt.html.md]: Change to "the oVirt Engine:"
Apologies for delay.

**Comment by petrkotas** [comment]
@mykaul All remarks are now fixed. Sorry for the delay. I got lost in the github notifications.

**Pull request merged**